src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -512,6 +512,14 @@ private void InitialHandshake()
                     //The first time we start replication we need to register the destination current CV
                     case ReplicationMessageReply.ReplyType.Ok:
                         LastAcceptedChangeVector = response.Reply.DatabaseChangeVector;
+                        // AllowedPaths will have a locally provided value for filtered pull replication.
+                        // In that case, we prefer the local value over the one from the remote server
+                        if (AllowedPaths == null)
+                        {
+                            // this is used when the other side lets us know what 
+                            // paths it is going to accept from us
+                            AllowedPaths = response.Reply.AllowedPaths;
```

Comment:
Need to validate that changing the definition on the Hub will be reflected on the already opened connections

Reply:
Right now, IIRC, that wouldn't be the case for things like removing certificates.
Existing connections are maintained. 

But I agree: https://issues.hibernatingrhinos.com/issue/RavenDB-15090